app/views/users/show.html.erb

```diff
@@ -1,16 +1,68 @@
-<h1> 
-<% if @user.name && @user.name!="" %>
-<%= @user[:name] + "'s Profile" %> </h1>
-<% else %>
-<%= @user[:email] + "'s Profile" %> </h1>
-<% end %>
-<div> <%= "Email: " + @user[:email] %> </div>
-<% if @user.payment == nil %>
-  <p>No preferred payment method listed</p>
-<% else %>
-<div> <%= "Preferred payment method: " + @user[:payment] %> </div>
-<%end %>
-
-
-<%= link_to("Back to item list", items_path, :class => "btn btn-primary") %>
-
+<div class="rounded bg-white mt-5">
+  <div class="body-container">
+    <div class="row">
+        <div class="col-md-4 border-right">
+            <div class="d-flex flex-column align-items-center text-center p-3 py-5">
+                <h1> <% if @user.name && @user.name!="" %>
+                    <%= @user[:name] + "'s Profile" %> </h1>
+                  <% else %>
+                    <%= @user[:email] + "'s Profile" %> </h1>
+                  <% end %>
+                  <div class="user-profiles">
+                      <% if @user[:id] == current_user.id %>
+                        <%= link_to("Edit profile", edit_user_path(current_user.id), :class => "btn btn-primary") %>
+                        <!--<a href='#fave-items'><button class="btn btn-primary">Go to my favorites</button></a>-->
```

Comment:
Will auto-scroll down to user's favorites (looks the same as unlisted/listed objects in screenshot above) once Noah's PR is merged